templates/profile.html

```diff
@@ -18,6 +18,16 @@ <h2>User profile</h2>
             <input type="email" class="form-control" name="email" id="email" minlength="3" maxlength="100" placeholder="E-mail" value="{{ .ProfileUser.Email }}">
         </div>
         <br>
+        <div class="form-label">
```

Comment:
Can you also put this on the other side of the .ShowForm branch so that NumTopics and NumComments are publicly visible?

Reply:
updated